config_models/admin.py

```diff
@@ -92,7 +92,7 @@ def revert(self, request, queryset):
         """
         if queryset.count() != 1:
             self.message_user(request, _("Please select a single configuration to revert to."))
-            return
+            return True
```

Comment:
it was giving quality warning i`nconsistent-return-statements`

Reply:
`return None` might be marginally better here since it preserves the previous behavior, but I'm not sure it makes a difference in this case.